### ISSUE TITLE
Add Workout Counter Sensor

### ISSUE THREAD

**Issue by edwork** [edwork/homeassistant-peloton-sensor#46]
Add a sensor, set of sensors, or sensor with sub-attributes that counts how many workouts a user has taken. This could be useful to watch for upcoming milestones.

**Comment by edwork** [comment]
@Miablo can you take a look at this one?

**Comment by Miablo** [comment]
@edwork i think there is a bool value that can be read for is_milestone that can be used instead of a counter but let me look into it a bit more and see what I come up with

**Comment by RobertD502** [comment]
Just to add to the discussion: The total number of workouts can be grabbed from the `total_workouts` key from the response returned by the GetMe function, which I see this integration [is already calling](https://github.com/edwork/homeassistant-peloton-sensor/blob/9d6e3d46720724342c4357bfe5642dd7e365e1ad/custom_components/peloton/__init__.py#L67).

**Comment by Miablo** [comment]
yeah that would then require a manual milestone counter for each milestone we would want to alert the user about since my interpretation of the issue wasnt to just count workouts but to be able to trigger a "this is a milestone" for whatever fitness type

**Comment by RobertD502** [comment]
I may be recalling incorrectly, but doesn't that alert already show up for users within the peloton app (mobile, bike, tread) when in a session that is a milestone ride, run, etc.?
